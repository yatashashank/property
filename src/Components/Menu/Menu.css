.menu {
	height: 100%;
	width: 200px;
	border-right: 1px solid lightgray;
	overflow-y: auto;
	background-color: #FAFAFB;
}

.menuItem {
	color: inherit;
	font-size: 14px;
	width: 100%;
	text-align: center;
	box-sizing: border-box;
	padding: 5px;
}

.menuItem:hover {
	color: #4282ad;
}