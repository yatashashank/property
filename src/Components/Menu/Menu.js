import React, { Component } from 'react';
import "./Menu.css";

class Menu extends Component {

    render() {
        if (this.props.show) return null;
        return (
            <div className="menu">
            

            </div>


        );
    }
}
export default Menu;
