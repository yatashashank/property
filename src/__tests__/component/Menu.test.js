import React from 'react';
import toJson from 'enzyme-to-json';

import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

import { shallow,render,mount } from 'enzyme';

import Menu from '../../Components/Menu/Menu'

describe('<Menu />', () => {

  it('snapshot testing for Menu component', () => {
    const wrapper = shallow(<Menu />);
    expect(toJson(wrapper)).toMatchSnapshot()
  });

  it('checking class name', () => {
    const wrapper = shallow(<Menu />);
    const tree=wrapper.find('.menu')
    expect(tree.length).toBe(1)
  });

  it('checking incoming props',()=>{
      const wrap=mount(<Menu show={true}/>);
      const val=wrap.props().show
      expect(val).toBeTruthy()
  })

})