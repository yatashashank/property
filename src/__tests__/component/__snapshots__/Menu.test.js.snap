// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Menu /> snapshot testing for Menu component 1`] = `
<div
  className="menu"
/>
`;
